src/main/scala/com/fullcontact/interview/RecordFinder.scala

```diff
@@ -1,7 +1,71 @@
 package com.fullcontact.interview
 
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.{DataFrame, SaveMode, SparkSession}
+
 object RecordFinder {
+  val Identifier = "identifier"
+  val Query = "query"
+  val Record = "record"
+  val Records = "records"
+
   def main(args: Array[String]): Unit = {
-    // TODO: Implement your job here
+    val spark = SparkSession.builder()
+      .master("local[4]")
+      .getOrCreate()
+
+    val records = spark.read.csv("./Records.txt").toDF(Record).repartition(4)
+    val queries = spark.read.csv("./Queries.txt").toDF(Query).repartition(4)
```

Comment:
Was there any particular reason for repartitioning with 4 partitions? If this was a much larger data set, might partitioning change? If we weren't running locally, would partitioning change?